parse/circleci/orb.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         type: string
         default: "-1"
       exclude:
-        description:  A regular expression defining the files and directories which the flag parser should exclude.
+        description:  "A regular expression (PCRE) defining the files and directories which the flag parser should exclude. Example: `vendor/`"
```

Comment:
ok, I see that you added the regex standard here. Based on your example, it looks like partial matches are filtered. just to be very explicit, we should say that (unless it actually does a prefix match, in which case we should say that